tests/distributed/test_pipeline_parallel.py

```diff
@@ -104,6 +104,7 @@ def iter_params(self, model_name: str):
 GENERATION_MODEL_SETTINGS = {
     # [DETAILED TESTS]
     "meta-llama/Meta-Llama-3-8B": PPTestSettings.detailed(),
+    "microsoft/Phi-3-mini-4k-instruct": PPTestSettings.detailed(trust_remote_code=True), # noqa: E501
```

Comment:
@youkaichao Are these the 2-node tests you were referencing?

I didn't see anywhere that multi-node was ran in CI, but I did verify that I can run this updated test locally with `VLLM_MULTI_NODE=1` after the fix.

Reply:
see https://github.com/vllm-project/vllm/blob/9a94ca4a5d31c0ba57ca67fc1c252233d3284012/.buildkite/test-pipeline.yaml#L363

Reply:
I think only these models are tested in the ci:

https://github.com/vllm-project/vllm/blob/9a94ca4a5d31c0ba57ca67fc1c252233d3284012/tests/distributed/test_pipeline_parallel.py#L198-L209

Reply:
cc @DarkLight1337  if we can add a test only for multi-node case.

Reply:
Sure, feel free to do so!

Reply:
I mean, I don't know how to add a test that's only executed in multi-node case 🤣  I'm new to this test config.

Reply:
I think you can just check the environment variable? Those configs are just to set the test parameters - multi-node was never a test param.

Reply:
I have updated the file so now you can set `multi_node_only` per `PPTestSettings`.